ansible/roles/docker/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 # role variable for the swarm service
 
 docker_api_port: 2385
+thin_provisioned: false 
```

Comment:
yes, I understand that vars can be defined at multiple places. But I started keeping all vars in `defaults` so there is just one way of introducing new vars. Otherwise it is hard to set a guideline for adding vars especially when a var is going to have a default value. I moved all vars to `defaults` for other reasons described in https://github.com/contiv/ansible/issues/61.